net/src/memory_worker.rs

```diff
@@ -58,34 +67,59 @@ impl NetWorker for MockWorker {
         Ok(())
     }
 
-    /// Set network's name as worker's endpoint
+    /// Set server's name as worker's endpoint
     fn endpoint(&self) -> Option<String> {
-        Some(self.network_name.clone())
+        Some(self.server_name.clone())
     }
 }
 
-impl MockWorker {
-    /// create a new mock worker... no configuration required
-    pub fn new(handler: NetHandler, network_config: &JsonString) -> NetResult<Self> {
-        let config: serde_json::Value = serde_json::from_str(network_config.into())?;
-        let network_name = config["networkName"]
+impl InMemoryWorker {
+    /// create a new memory worker connected to an in-memory server
+    pub fn new(handler: NetHandler, backend_config: &JsonString) -> NetResult<Self> {
+        // Get server name from config
+        let config: serde_json::Value = serde_json::from_str(backend_config.into())?;
+        // println!("InMemoryWorker::new() config = {:?}", config);
+        let server_name = config["serverName"]
             .as_str()
             .unwrap_or("(unnamed)")
             .to_string();
-
-        let mut map_lock = MOCK_MAP.write().unwrap();
-        if !map_lock.contains_key(&network_name) {
-            map_lock.insert(network_name.clone(), Mutex::new(MockSystem::new()));
+        println!("InMemoryWorker::new() server_name = {}", server_name);
```

Comment:
What's our logging strategy these days? There's a lot of `println!`s, either active or commented out in here. It'd be nice to have tagged logging statements at various levels that could all always be in here, but configure-time enabled / disabled.

Reply:
Those printfs are temporary. I'll remove them once the PR is green.
As for logging strategy, 'core' has its own logger system, but I don't think 'net' should be dependent on core. I'm thinking of rolling our own in 'net', but no steps have been made in that direction yet.